ext_mongodb.php

```diff
@@ -210,7 +210,7 @@ function($value) {
 <<__NativeData("MongoDBDriverManager")>>
 class Manager {
 	<<__Native>>
-	public function __construct(string $dsn = "mongodb://127.0.0.1/", array $options = array(), array $driverOptions = array());
+	public function __construct(string $dsn = "", array $options = array(), array $driverOptions = array());
```

Comment:
Passing an empty string to mongoc btw, makes it return a "can not parse URL" error - that's how I found this in the first place.